student_class.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(name, age, parent_permission, classroom)
+    super(name, age, parent_permission)
+    @classroom = classroom
+  end
+
```

Comment:
- [ ] Conider adding a relation between student and classroom( student belongs to a classroom). Talke alook for this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) to know how can to do it.